### ISSUE TITLE
mobs and animals not found

### ISSUE THREAD

**Issue by RayPS** [PocketMine/PocketMine-MP#2630]
This is my server.properties

```
#Properties Config file
#Thu Feb 12 04:44:22 UTC 2015
type=0
server-name=Minecraft:PE Server
server-port=19132
memory-limit=256M
gamemode=0
max-players=20
spawn-protection=-1
white-list=off
enable-query=on
enable-rcon=off
motd=Minecraft: Welcome!
announce-player-achievements=on
allow-flight=off
spawn-animals=on
spawn-mobs=on
force-gamemode=off
hardcore=off
pvp=on
difficulty=1
generator-settings=
level-name=world
level-seed=
level-type=DEFAULT
rcon.password=TyQpoj0QWW
auto-save=on
```

Can't find any animals and monsters, game mode is survival.
Everytime i try to delete the "world" folder and regenerate the map it's always looks similar.

**Comment by flaxues** [comment]
Animals and monsters aren't working yet

**Comment by luca28pet** [comment]
Plus, biomes aren't implemented in 1.4, so the only biome you can find is the default one.
